support-frontend/assets/pages/promotion-terms/weeklyTerms.tsx

```diff
@@ -50,11 +49,10 @@ const orderedCountryGroupNames: NameAndSaving[] = [
 		saving: null,
 	},
 ];
-const shortTermDescription =
-	postIntroductorySixForSixBillingPeriod === Quarterly
-		? 'Quarterly (13 weeks)'
-		: 'Monthly (4 weeks)';
-export default function WeeklyTerms(props: PromotionTermsPropTypes) {
+const shortTermDescription = 'Monthly (4 weeks)';
```

Comment:
Could actually inline this now that it doesn't vary any more. Up to you though.

Reply:
I'm going to keep it as a variable as it's referenced in 2 places in the code.